dll/keyboard/kbdit/kbdit.c

```diff
@@ -155,9 +155,6 @@ ROSDATA USHORT scancode_to_vk[] = {
 /* 7c */  VK_EMPTY,
 /* 7d */  VK_EMPTY,
 /* 7e */  VK_EMPTY,
-/* 7f */  VK_EMPTY,
```

Comment:
Interesting that it stops at 7e, and 7f is not even "empty" xd

Reply:
Yeah it's just array of values, so zero-termination is not needed here :slightly_smiling_face: